.dockerignore

```diff
@@ -1,13 +1,20 @@
-**/node_modules/
-**/node_modules_linux/
+**/.circleci/
+**/.vscode/
+**/.idea/
+**/coverage/
 **/docs/
-**/Dockerfile
+**/node_modules/
+**/node_modules_linux/g
 **/.dockerignore
 **/.editorconfig
-**/.eslintrc.js
-**/.gitignore
 **/.git
-**/README.md
+**/.gitattributes
+**/.gitignore
+**/.travis.yml
+**/CODE-OF-CONDUCT.md
+**/Dockerfile
```

Comment:
This seems like something you'd want to include, not ignore?

Reply:
Since the Docker uses the Dockerfile to build the image, the image doesn't need to carry the plan with itself. So ignoring it makes the image smaller.

Reply:
OK, sounds good.